run_tests.py

```diff
@@ -153,7 +153,7 @@ def is_cross_build(self):
     print('Running unittests.\n')
     units = ['InternalTests', 'AllPlatformTests']
     if mesonlib.is_linux():
-        units += ['LinuxlikeTests']
+        units += ['LinuxlikeTests', 'LinuxArmCrossCompileTests']
```

Comment:
These cross tests should only be done if the cross toolchain is installed. In `run_tests` it is done like this (at the end of the file):

    shutil.which('arm-linux-gnueabihf-gcc-6') and not platform.machine().startswith('arm')

This should be extracted to a function `should_run_linux_cross_tests()` and used from both locations.